src/upgrade/db.ts

```diff
@@ -0,0 +1,97 @@
+import * as hre from "hardhat";
+import { MongoDBAdapter } from "../deploy/db/mongo-adapter/mongo-adapter";
+import { VERSION_TYPES } from "../deploy/db/mongo-adapter/constants";
+import { getLogger } from "../deploy/logger/create-logger";
+import { znsNames } from "../deploy/missions/contracts/names";
+import { getContractDataForUpgrade } from "./upgrade";
+
+
+export const updateAllContractsInDbAndVerify = async ({
+  dbAdapter,
+} : {
+  dbAdapter : MongoDBAdapter;
+}) => {
```

Comment:
using this function parameter syntax with `{ ...args } : { ...argTypes }` feeels a bit clunky when only a single parameter is used, the whole "parameter ordering" argument can't apply here either